pytorch_lightning/accelerators/gpu.py

```diff
@@ -53,6 +64,57 @@ def set_nvidia_flags(local_rank: int) -> None:
         devices = os.getenv("CUDA_VISIBLE_DEVICES", all_gpu_ids)
         _log.info(f"LOCAL_RANK: {local_rank} - CUDA_VISIBLE_DEVICES: [{devices}]")
 
+    def get_device_stats(self, device: Optional[torch.device] = None) -> None:
```

Comment:
```suggestion
    def get_device_stats(self, device: Optional[torch.device] = None) -> Dict[str, Any]:
```

Reply:
@daniellepintz We should be returning logs here, right?